thirdeye-persistence/src/test/java/ai/startree/thirdeye/datalayer/bao/TestAnomalyManager.java

```diff
@@ -206,6 +237,12 @@ public void testSaveChildren() {
     Assert.assertNotNull(child2.getId());
   }
 
+  private AnomalyDTO buildAnomaly(long startTime, long endTime) {
```

Comment:
This can be `static` right? There is an existing method `anomalyWithCreateTime`. We can have them in the same place. Please use the intellij format to rearrange the code so you can find the util methods. by default static methods are at the top.

Reply:
Yes, will do.